src/formatDate.js

```diff
@@ -50,7 +50,41 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const onlyDateElements = 2;
+  const lastElement = fromFormat.length - 1;
+  const obj = {};
+  const arrDate = date.split(`${fromFormat[lastElement]}`);
```

Comment:
fromFormat[lastElement] - string

```suggestion
  const arrDate = date.split(fromFormat[lastElement]);
```